auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java

```diff
@@ -307,52 +309,48 @@ private void signingWithCreds(@NonNull PhoneAuthCredential phoneAuthCredential)
         AuthInstances.getFirebaseAuth(getFlowParams())
                 .signInWithCredential(phoneAuthCredential)
                 .addOnSuccessListener(this, new OnSuccessListener<AuthResult>() {
-            @Override
-            public void onSuccess(final AuthResult authResult) {
-                mVerificationState = VerificationState.VERIFIED;
-                completeLoadingDialog(getString(R.string.verified));
-
-                // Activity can be recreated before this message is handled
-                mHandler.postDelayed(new Runnable() {
                     @Override
-                    public void run() {
-                        if (!mIsDestroyed) {
-                            dismissLoadingDialog();
-                            finish(authResult.getUser());
-                        }
+                    public void onSuccess(final AuthResult authResult) {
+                        mVerificationState = VerificationState.VERIFIED;
+                        completeLoadingDialog(getString(R.string.verified));
+
+                        // Activity can be recreated before this message is handled
+                        mHandler.postDelayed(new Runnable() {
+                            @Override
+                            public void run() {
+                                if (!mIsDestroyed) {
+                                    dismissLoadingDialog();
+                                    finish(authResult.getUser());
+                                }
+                            }
+                        }, SHORT_DELAY_MILLIS);
                     }
-                }, SHORT_DELAY_MILLIS);
-            }
-        }).addOnFailureListener(this, new OnFailureListener() {
+                }).addOnFailureListener(this, new OnFailureListener() {
             @Override
```

Comment:
nit: this indentation is strange, the onFailure method is to the left of addOnFailureListener